pytorch/inference/docker/build_artifacts/torchserve-entrypoint.py

```diff
@@ -15,8 +15,29 @@
 import shlex
```

Comment:
@chen3933 does this also need to be added to the ec2 torchserve entrypoint? https://github.com/aws/deep-learning-containers/blob/master/pytorch/inference/docker/build_artifacts/torchserve-ec2-entrypoint.py

Reply:
On EC2 customer can run 
```
docker run --init <image_name>
```
which will enable pid 1 to reap zombie child processes.